singularity/Singularity.amd_aocl

```diff
@@ -285,7 +285,19 @@ From: fedora:41
     pip install "numpy==1.26.2"
     pip install --no-binary pandas 'pandas==2.0.0'
     pip install --no-binary h5py h5py
-    pip install -r $INSTALLDIR/requirements3.txt
+
+    # temporary fix for https://github.com/tikk3r/flocs/issues/100#issuecomment-2543192437
```

Comment:
I'm already looking into this and I think installing directly from github works, so I think this isn't needed.

Reply:
Oh apparently not, it looked like it did but trying again it doesn't.

Reply:
There are two ways of fixing it now. The one I put above (using `sed`) and:
https://github.com/ratt-ru/shadeMS/issues/124#issuecomment-2544217464
Both work for me.